gen_mod_headers

```diff
@@ -183,6 +183,11 @@ for f in $(find . -name "Kconfig*"); do
 	copy_mkdir "$target_dir" "$f"
 done
 
+# Fix arm64 file reference
+if [ $karch = "arm64" ] && [ -e $karch_dir/include/asm/opcodes.h ]; then
+	copy_mkdir "$target_dir" "./arch/arm/include/asm/opcodes.h"
```

Comment:
Looking at `copy_mkdir`, the relative path as `arch/arm/include/asm/opcodes.h` would be better? (instead of `./arch/...`

Reply:
Also, the test and the copy is not clear if they are the same or different files. If the same, can you do the test and copy with the same path?